LinqToRest/ResourceRetriever.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;

```

Comment:
Cleanup functions?

Reply:
Addressed in https://github.com/messerli-informatik-ag/linq-to-rest/pull/10/commits/f597f32ca9af08d1f52582ac308e5c9c0853b2c3, https://github.com/messerli-informatik-ag/linq-to-rest/pull/10/commits/5b64ed9b05cb82f4254a44964b9adafeb6d43d69 and https://github.com/messerli-informatik-ag/linq-to-rest/pull/10/commits/49cf9a9be4fa79c4f240dff62fa65692706dc843